features/share/graphql/creating-our-uni.md

```diff
@@ -0,0 +1,46 @@
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://raw.githubusercontent.com/vendia/examples/main/vendia-logo.png" alt="vendia logo" width="100px">
+  </a>
+</p>
+
+# Creating Our Universal Application
+
+## Pre-requisites
+
+* [Vendia Share CLI](https://vendia.net/docs/share/cli)
```

Comment:
```suggestion
* Install the [Vendia Share CLI](https://vendia.net/docs/share/cli)
```

Reply:
Preference is to keep it as-is. It's consistently used throughout the examples repo. I can change it if you feel strongly but don't think it's much of a difference.